packages/next-server/lib/side-effect.js

```diff
@@ -37,6 +37,7 @@ export default function withSideEffect (reduceComponentsToState, handleStateChan
       static canUseDOM = typeof window !== 'undefined'
 
       static contextTypes = WrappedComponent.contextTypes
```

Comment:
I think this can be removed

Reply:
Depends on how we choose to consume the context in `head.js`, see my comment above

Reply:
Ops nvm, read the wrong line 😛